src/java/org/apache/cassandra/index/sai/disk/vector/CassandraOnHeapGraph.java

```diff
@@ -351,6 +352,7 @@ public SegmentMetadata.ComponentMetadataMap flush(IndexDescriptor descriptor, Se
         try (var pqOutput = IndexFileUtils.instance.openOutput(descriptor.fileFor(IndexComponent.PQ, context), true);
              var postingsOutput = IndexFileUtils.instance.openOutput(descriptor.fileFor(IndexComponent.POSTING_LISTS, context), true);
              var indexWriter = new OnDiskGraphIndexWriter.Builder(builder.getGraph(), indexFile.toPath())
+                               .withVersion(2) // always write old-version format since we're not using the new features
```

Comment:
Is it worth defining these `2` values as a static constant somewhere so we know all of the references? Or will we be removing this pretty soon?